src/textual/app.py

```diff
@@ -248,17 +247,37 @@ class App(Generic[ReturnType], DOMNode):
         ```python
         self.app.dark = not self.app.dark  # Toggle dark mode
         ```
+    """
+
+    _devtools_port: int = constants.DEFAULT_DEVTOOLS_PORT
```

Comment:
I don't think any of these need to be classvars.

Any reason why they shouldn't be instance variables?

Reply:
Not really, no.